File: C++/covenant-university/v4/demo.v4.cpp
/* Library Files */
    // Input-Output Stream
    #include <iostream>

    // Lapys Header
    #include "lapys (2018 - Q4).h"

/* Utilities */
    // Namespace > Standard Library
    using namespace std;

// Function
    // Power
    double power(double x, int p) {
        // Initialization > Multiplier
        double multiplier = x;

        // Loop
        while (p != 0) {
            // Update > (...)
            x *= multiplier;
            p -= 1;
        }

        // Return
        return x;
    };

    /* Main */
    int main() {
        // Initialization > (...)
        double x = 5;
        int p = 2;

        // Print
        print("Power of (", x, ", ", p, "): ", pow(x, p));

        // Return
        return 0;
    }

File: Technologies/Python/Covenant University/v2/demo.v4.py
# Initialization > (Denominator, Numerator, Total)
denominator = 30
numerator = 1
total = 0

# Loop
while denominator != 0 and numerator != 31:
    # Update > Total
    total += numerator / denominator

    # Update > (Denominator, Numerator)
    denominator -= 1
    numerator += 1

# Print
print('\n\nTotal:', total)
